include/std_filesystem.h

```diff
@@ -0,0 +1,32 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2021-2021  The DOSBox Staging Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#ifndef DOSBOX_FILESYSTEM_H
+#define DOSBOX_FILESYSTEM_H
+
+#if defined(__clang__) || defined(_MSC_VER) || (defined(__GNUC__) && __GNUC_ >= 9)
```

Comment:
Looks like a small typo here because of which GCC version detection would always fail:
```suggestion
if defined(__clang__) || defined(_MSC_VER) || (defined(__GNUC__) && __GNUC__ >= 9)
```